apps/www/src/components/layout/index.tsx

```diff
@@ -14,16 +15,18 @@ type LayoutProps = {
  */
 export async function PageLayout({ children }: LayoutProps) {
   const { isNativeApp } = getPlatformInformation()
+  const me = await getMe()
 
   return (
     <div
       className={css({
+        backgroundColor: 'pageBackground',
         minHeight: '100dvh',
         display: 'flex',
         flexDirection: 'column',
       })}
     >
-      <PageHeader />
+      <PageHeader me={me} />
```

Comment:
This will add a bunch more data to the client which isn't really needed. It would be better to specify which props are actually needed in the PageHeader and only pass those from the server to the client.